main/acle_pdf_conf.yaml

```diff
@@ -0,0 +1,16 @@
+title: Arm C Language Extensions
+subtitle: 2021Q2
+date: 02 July 2021
+logo: ./tools/Arm-logo-blue-RGB.pdf
+documentclass: article
+toc: true
+numbersections: true
+geometry: landscape,top=3cm,bottom=3cm,left=2cm,right=2cm
```

Comment:
This line is wrong. It should be the same used for the morello one. In particular, the `landscape` layout is reserved for the specs with the long list of intrinsics.

Reply:
I realised I hadn't changed it according to the TechComms example